webapp/pages/profile/_id/_slug.vue

```diff
@@ -507,7 +459,11 @@ export default {
         return UserQuery(this.$i18n)
       },
       variables() {
-        return { id: this.$route.params.id }
+        return {
+          id: this.$route.params.id,
+          followedByCount: this.followedByCount || 7,
+          followingCount: this.followingCount || 7,
```

Comment:
do we need to have both a default in `data` and a fallback here?

Reply:
no, we actually don't
removed in 9dd65aa